beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/performance/SyncCommitteePerformanceTracker.java

```diff
@@ -201,7 +202,7 @@ private synchronized int countIncludedMessages(
             slot);
         continue;
       }
-      for (Integer committeeIndex : committeeIndices) {
+      for (int committeeIndex : committeeIndices) {
```

Comment:
I don't think this is taking advantage of the `IntSet` - it will create a `Iterator<Integer>` which wraps each value and then the value is unwrapped back to `int`.  We'd need to use `IntIterator` directly and call `nextInt()` rather than just `next()`. In this case it may be better to use an `IntStream` instead like:
```
numberOfIncludedMessages += committeeIndices.intStream().filter(syncCommitteeBits::getBit).count();
```

Reply:
Ah I didn't think of that. I think you're right. I'll change it to one of those.